dash-tx-value-signing.md

```diff
@@ -0,0 +1,250 @@
+<pre>
+DIP: tx-value-signing
+Title: Transaction value signing analogous to BIP143 as implemented in Bitcoin Cash
```

Comment:
This mentions Bitcoin Cash yet there is no link to the corresponding spec (https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/replay-protected-sighash.md ?) and in general it looks more like bip143 with segwit related parts removed.